ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -53,12 +62,14 @@ class_relationships:
 - Router 1:M FloatingIp
 - Router M:M Subnet
 - Hypervisor 1:1 Host
+# Cinder
+- Instance 1:M Volume
 device_classes: !ZenPackSpec
   /Server/SSH/Linux/NovaHost:
     zProperties:
       zCollectorPlugins: [zenoss.cmd.uname, zenoss.cmd.uname_a, zenoss.cmd.df, zenoss.cmd.linux.cpuinfo,
         zenoss.cmd.linux.memory, zenoss.cmd.linux.ifconfig, zenoss.cmd.linux.netstat_an,
-        zenoss.cmd.linux.netstat_rn, zenoss.cmd.linux.process, zenoss.cmd.linux.rpm,
+        zenoss.cmd.linux.netstat_rn, zenoss.cmd.linux.process, zenoss.cmd.linux.lvm, zenoss.cmd.linux.rpm,
```

Comment:
I believe you are introducing a dependency on an unreleased version of LinuxMonitor here.   We would need to add that dependency to setup.py.  But really, I think what this is pointing out is that we shouldn't be setting zCollectorPlugins this way in the first place.  

Can you open a jira to track this issue?  We really want to just append our plugins on top of whatever ones already exist, not to hardcode the list.  We can do this in python code if necessary.   

I'm ok with merging this as it is for now, but we need to fix this before we can release.